test/41-device-api-v1.spec.ts

```diff
@@ -298,11 +304,13 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 	describe('POST /v1/apps/:appId/start', () => {
 		it('does not allow starting an application when there is more than 1 container', async () => {
 			// Every test case in this suite has a 3 service release mocked so just make the request
```

Comment:
We can remove all instances of this comment since we now specify the containers (via testWithData) for all requests.

Reply:
nvm I see the tests which still have this comment aren't running with `testWithData`